test/manifest.ts

```diff
@@ -6908,7 +6908,7 @@ version = "3.0.0"
       );
     });
 
-    it('should build prerelease releases from pre-major', async () => {
+    it('should not build prerelease releases from pre-major', async () => {
```

Comment:
Do we have a test case that covers `0.x.y` AND the label `autorelease: pre-release`? To be sure users have a way to overwrite that behavior if they want a specific release to be tagged as pre-release.

Reply:
added a test